src/com/twitter/intellij/pants/service/project/PantsSystemProjectResolver.java

```diff
@@ -83,7 +83,9 @@
     task2executor.put(id, executor);
     final DataNode<ProjectData> projectDataNode =
       resolveProjectInfoImpl(id, executor, listener, isPreviewMode, settings.isEnableIncrementalImport());
-    doViewSwitch(id, projectPath);
+    if (!settings.isImportSourceDepsAsJars()) {
```

Comment:
(From anecdotal evidence, take with a grain of salt)
The users I've talked to said that even with the improved "Project" view that "Import Source Deps as Jars" brings, they'd still prefer the "Project File Tree" view to be the default, as it allows them to search for files that are excluded more easily.

Reply:
Gotcha. will revert.